python/orca/src/bigdl/orca/learn/pytorch/experimential/mmcv/mmcv_ray_runner.py

```diff
@@ -198,11 +198,19 @@ def validate(self, **kwargs):
 
     def get_state_dict(self):
         """Returns the state of the runner."""
-        pass
+        state = {
+            "epoch": self._epoch,
+            "model": self.model.module.state_dict(),
+            "optimizer": self.optimizer.state_dict()
+        }
```

Comment:
How do mmcv save scheduler(in their scheduler hooks)?

Reply:
mmcv use `LrUpdaterHook` to replace LR scheduler, and do not save scheduler info